README.md

```diff
@@ -112,7 +101,12 @@ The type of `formatted` will depend on the formatter function and can be safely
 
 Three formatters are included out of the box:
 
-* `SQLDatabaseFormatter` returns format to SQL technology such as PostgreSQL and MySQL.
+* `SQLDatabase` returns format to SQL technology such as PostgreSQL and MySQL.
+* `MongoDB` returns a MongoDB connection as MongoClient instance
+* `MySQL` returns a MySQL connection as DataSource.
+* `PostgreSQL` returns a PostgreSQL connection as DataSource.
+* `Redis` returns a Redis connection as JedisPool.
+* `RedisSpring` returns a Redis connection as JedisConnectionFactory.
```

Comment:
Same with the rest of these.  I'm unclear what's being returned here, but the implication is these formatters are doing more work than they should.